ocs_ci/ocs/must_gather/must_gather.py

```
@@ -251,15 +254,19 @@ def verify_noobaa_diagnostics(self):
-                        files_noobaa_diag.extractall(path)
-                        break
+            self.get_all_paths()
+            for full_path in self.full_paths:
+                if re.search(
+                    r"noobaa_diagnostics_.*.tar.gz", os.path.basename(full_path)
+                ):
+                    flag = True
+                    if os.path.isabs(full_path):
+                        logger.info(f"Extract noobaa_diagnostics: {full_path}")
+                        with tarfile.open(full_path) as f:
+                            f.extractall(os.path.dirname(full_path))
```

Ruff (0.15.17)

[error] Uses of `tarfile.extractall()`

(S202)